services/dy-dash/mattward-dash/src/Dockerfile

```diff
@@ -11,7 +11,9 @@ USER $NB_USER
 # install requirements --------------------------------------------------------
 COPY --chown=jovyan:users src/requirements.txt .
 # TODO: tmp workaround for https://github.com/pypa/pip/issues/6197
-RUN pip --cache-dir=/tmp/.pipcache install -r requirements.txt &&\
+ENV VIRTUAL_VENV=/home/jovyan/.venv
+RUN python -m venv ${VIRTUAL_VENV} &&\    
+    ${VIRTUAL_VENV}/bin/pip --cache-dir=/tmp/.pipcache install -r requirements.txt &&\
```

Comment:
why not using --no-cache instead of dropping cache in a tmp and remove it?

Reply:
this is a bit beyond the scope of just a bugfix.
for the version let me talk with @odeimaiz first to see if this will not break any template.

Reply:
also the image is already 10Gb, so I don't think the no-cache will bring much here. will add it though for conscience...